src/30_Advanced/AMP_cache.html

```diff
@@ -0,0 +1,56 @@
+<!--
+  #### Introduction
+
+  Google provides a cache that clients can use for free to display AMP files. This gives you two choices to display an AMP file in your app using either:
```

Comment:
We have to make the intro more generic - the bit you've used is specific for apps.